### PR TITLE
Add Sylius 1.10 support

### PR DESCRIPTION
**Pull request by oallain**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?   | no
| Fixed tickets | #263 

Add Sylius 1.10 support.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.3",
```
Comment (Jibbarth) [inline composer.json:7]: Is it possible to allow php8 ? :innocent:

Comment (mbabker) [reply composer.json:7]: Not without addressing https://github.com/FriendsOfSylius/SyliusImportExportPlugin/issues/260 at a minimum.

Comment (oallain) [reply composer.json:7]: Yes, if you feel like PR will be welcome.

**Comment by andriusvo** [comment]
@oallain can it be merged and new tag released?

**Comment by oallain** [comment]
@andriusvo it's done https://github.com/FriendsOfSylius/SyliusImportExportPlugin/issues/263#issuecomment-877329925

**Pull request merged**